### ISSUE TITLE
HTTPInternalServerError message not displayed

### ISSUE THREAD

**Issue by samuelcolvin** [aio-libs/aiohttp_debugtoolbar#45]
In the following screen shot aiohttp_jinja has tried really hard to provide a helpful message along with the 500 response however aiohttp_debugtoolbar isn't showing that message.

I guess a fix is as simple as including `exc.args`?

![image](https://cloud.githubusercontent.com/assets/4039449/18783713/50ac55f0-8185-11e6-9b7b-512a987d78bd.png)

**Comment by asvetlov** [comment]
@jettify could you take a look?

**Comment by jettify** [comment]
I will take a look

**Comment by jettify** [comment]
I investigated this issue, toolbar extracts error message, which is `Internal Server Error`, text that was generated from `jinja2` is not exception message but payload for response with 500 status code. In order have nice message in `aiottp_jinja2` we should do following
```python
    try:
        template = env.get_template(template_name)
    except jinja2.TemplateNotFound as e:
        msg = "Template '{}' not found".format(template_name)
        raise web.HTTPInternalServerError(text=msg, reason=msg) from e
```

**Comment by asvetlov** [comment]
Please make a PR for aiohttp_jinja2

**Comment by samuelcolvin** [comment]
I guess this is closed since it's agreed it's caused by a problem on aiohttp_jinja2 rather than here and the PR on that package is merged.

**Comment by asvetlov** [comment]
Yes, it's fixed by aiohttp_jinja2==0.9.0
